definitions/npm/@ginkgo-bioworks/react-json-schema-form-builder_v1.x.x/flow_v0.30.x-v0.103.x/react-json-schema-form-builder_v1.x.x.js

```diff
@@ -0,0 +1,120 @@
+declare module '@ginkgo-bioworks/react-json-schema-form-builder' {
+  declare type CardProps = {
+    name: string,
+    required: boolean,
+    dataOptions: { [string]: any },
+    uiOptions: { [string]: any },
+    $ref?: string,
+    dependents?: Array<{
+      children: Array<string>,
+      value?: any,
+    }>,
+    dependent?: boolean,
+    parent?: string,
+    propType: string,
+    neighborNames: Array<string>,
+  };
```

Comment:
From Flow 0.83.x all object types must be explicitly marked as either exact or inexact.

Reply:
thanks for the heads up, marking everything as exact in next commit